internal/storage/v2/elasticsearch/tracestore/fixtures/es_02.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
the convention in the fixtures is that es_01 corresponds to otel_01, es_02 to otel_02, etc. Don't break this convention by introducing arbitrary naming scheme. If you have two different flavors of ES span (e.g. one with materialized fields another without) then they should still be named es_01_{suffix}